man/systemd.generator.xml

```diff
@@ -186,6 +189,17 @@
           </para>
         </listitem>
 
+        <listitem>
+          <para>
+            Units created by generators to /run/systemd/system might not be 
+            written to the root of that directory; they may exist in a
+            subdirectory. Nevertheless, desired vendor overrides for
+            generated unit files should be placed directly in
+            /etc/systemd/system as described in
+            <citerefentry><refentrytitle>systemd.unit</refentrytitle><manvolnum>5</manvolnum></citerefentry>.
```

Comment:
... and then this paragraph can be deleted, since it doesn't add anything new. I'd rather not extend the notes list. It's there only because this man page was created from a wiki page that had this format, but this is far from ideal.

Reply:
I'll see what I can do. The fact that you do see the clear need FOR this pull is a help in that regard.